libs/wingsdk/src/target-sim/redis.inflight.ts

```diff
@@ -24,7 +24,7 @@ export class Redis implements IRedisClient, ISimulatorResourceInstance {
   public constructor(_props: RedisSchema["props"], context: ISimulatorContext) {
     this.context = context;
     this.container_name = `wing-sim-redis-${this.context.resourcePath.replace(
-      "/",
+      /\//g,
       "."
     )}-${uuidv4()}`;
```

Comment:
Can we create the name using the name generating utility we have instead? https://github.com/winglang/wing/blob/main/libs/wingsdk/src/utils/resource-names.ts

Reply:
😁 @Chriscbr I was just thinking about this over the weekend when I was playing with some things. We should definitely use that utility. Ill update that since Im already playing in the redis stuff.

Reply:
:+1:  til